packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+# Color
+FAST utilizes an adaptive color system that provides some unique advantages:
+1. Ensure text meets [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) contrast requirements.
+2. Easily swap from light mode to dark, or anywhere in-between.
+3. Color themeing through palette tinting.
+4. Perceptually uniform UI across different surface colors.
+
+## Algorithmic colors
```

Comment:
You use the word "recipe" multiple times below, either add the word "recipe" to the title here or the body text underneath so the reader can draw the conclusion that an algorithm that generates a color is a recipe?